pkg/common/utils/chunk.go

```diff
@@ -57,6 +57,7 @@ func ParseChunkSize(r network.Reader) (int, error) {
 	if err != nil {
 		return -1, errors.NewPublic(fmt.Sprintf("cannot read '\n' char at the end of chunk size: %s", err))
 	}
+	// todo support chunk extension
```

Comment:
maybe to do can remove first. I think it's not a common feat. If some user has the requirement, we can support it.